frontend/src/scenes/onboarding/productAnalyticsSteps/DashboardTemplateVariables.tsx

```diff
@@ -0,0 +1,186 @@
+                        <LemonInput
+                            className="grow"
+                            onChange={(v) => {
+                                if (v) {
+                                    setCustomEventName(v)
+                                    setVariable(variable.name, {
+                                        events: [{ id: v, math: 'dau', type: 'events' }],
+                                    })
+                                } else {
+                                    setCustomEventName(null)
+                                    resetVariable(variable.id)
+                                }
+                            }}
+                            onBlur={() => {
+                                if (customEventName) {
+                                    setVariable(variable.name, {
+                                        events: [{ id: customEventName, math: 'dau', type: 'events' }],
+                                    })
+                                } else {
+                                    resetVariable(variable.id)
+                                    setShowCustomEventField(false)
+                                }
+                            }}
+                        />
+                        <div>
+                            <LemonButton
+                                icon={<IconTrash />}
+                                type="tertiary"
+                                size="small"
+                                onClick={() => {
+                                    resetVariable(variable.id)
+                                    setCustomEventName(null)
+                                    setShowCustomEventField(false)
+                                }}
+                            />
+                        </div>
+                    </div>
+                </div>
+            )}
+            {!hasSelectedSite ? (
+                <LemonBanner type="warning">Please select a site to continue. </LemonBanner>
+            ) : (
+                <div className="flex">
+                    {variable.touched ? (
+                        <>
+                            {!allVariablesAreTouched ||
+                            (allVariablesAreTouched && variables.length !== activeVariableIndex + 1) ? (
+                                <LemonButton
+                                    type="primary"
+                                    status="alt"
+                                    onClick={() =>
+                                        !allVariablesAreTouched
+                                            ? goToNextUntouchedActiveVariableIndex()
+                                            : variables.length !== activeVariableIndex + 1
+                                            ? incrementActiveVariableIndex()
+                                            : null
+                                    }
+                                >
+                                    Continue
```

Comment:
I feel like we need a skip incase they give up

Reply:
Hm that's a good point, will add